File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdatePlantsTagRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantsUpdatePlantsTagRequestItem {
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("Label")
    public String label;
    @JsonProperty("NewTag")
    public String newTag;
    @JsonProperty("ReplaceDate")
    public String replaceDate;
    @JsonProperty("TagId")
    public Integer tagId;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdatePlantsTagRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsUpdatePlantsTagRequestItem(
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("NewTag")
    val newTag: String? = null,
    @JsonProperty("ReplaceDate")
    val replaceDate: String? = null,
    @JsonProperty("TagId")
    val tagId: Int? = null
)
